src/test/java/io/strimzi/kafka/bridge/http/ConsumerGeneratedNameTest.java

```diff
@@ -50,18 +58,23 @@ void createConsumerNameNotSet(VertxTestContext context) {
                         assertEquals(HttpResponseStatus.OK.code(), response.statusCode());
                         JsonObject bridgeResponse = response.body();
                         String consumerInstanceId = bridgeResponse.getString("instance_id");
+                        name.set(consumerInstanceId);
                         assertTrue(consumerInstanceId.startsWith("kafka-bridge-consumer-"));
+                        create.complete(true);
                     });
-                    context.completeNow();
                 });
+        create.get(TEST_TIMEOUT, TimeUnit.SECONDS);
+        consumerService().deleteConsumer(context, groupId, name.get());
```

Comment:
let's use the same format as for other tests, putting the method call on the next line.